src/js/containers/covid19/recipient/SummaryInsightsContainer.jsx

```diff
@@ -74,16 +74,20 @@ const SummaryInsightsContainer = ({ activeFilter }) => {
         }
         awardAmountRequest.current = fetchAwardAmounts(params);
         recipientCountRequest.current = fetchDisasterSpendingCount('recipient', params);
-
-        awardAmountRequest.current.promise
-            .then((res) => {
-                setAwardObligations(res.data.obligation);
-                setAwardOutlays(res.data.outlay);
-                setNumberOfAwards(res.data.award_count);
-            });
-        recipientCountRequest.current.promise
-            .then((res) => {
-                setNumberOfRecipients(res.data.count);
+        Promise.all([
```

Comment:
Do we want to use `Promise.all`? This will change the functionality not updating the totals as they are received from the API. Did we consult design or the PO's on this decision?